yaml/examples/sample_deployment.yaml

```diff
@@ -0,0 +1,43 @@
+kind: Deployment
```

Comment:
why daemonset and Deployment?

Reply:
i thought i would have an example of every pod resource, so that when i need a pod, i would find a pod, when i need deployment i will find one, and when i need a daemonset i would find one.

Reply:
it is just an example, it is not used by the scripts